pages/instructorQuestionManualGrading/instructorQuestionManualGrading.sql

```diff
@@ -0,0 +1,19 @@
+-- BLOCK instance_question_select_last_variant 
+SELECT v.*
+FROM
+    variants as v
+    JOIN instance_questions as iq ON (v.instance_question_id = iq.id)
+WHERE instance_question_id = $instance_question_id
+ORDER BY v.id DESC
+LIMIT 1;
+
+-- BLOCK instance_question_select_question
+SELECT q.*
+FROM
+    variants as v
+    JOIN instance_questions as iq ON (v.instance_question_id = iq.id)
+    JOIN submissions as s ON (v.id = s.variant_id)
+    JOIN assessment_questions as aq ON (iq.assessment_question_id = aq.id)
+    JOIN questions as q ON (aq.question_id = q.id)
+WHERE instance_question_id = $instance_question_id
+ORDER BY v.id DESC;
```

Comment:
Should this be ordered by ID or time submitted?

Reply:
Technically both. I have to find the exact convention, but I believe that it is Date first, and then Id second. Good question to ask why that order in the meeting, as Matt probably knows the exact subtle reason why. I saw an example where it was ordered by an ID and date. Cannot remember the exact date field, but I will try to find it.

Reply:
I don't know what I was thinking when I wrote this query. Completely changed this.

Only one question will be queried based on a question_instance_id so there will be no sorting.